modules/seqexec/server/src/main/scala/seqexec/server/gpi/GpiHeader.scala

```diff
@@ -25,7 +25,7 @@ object GpiHeader {
         val ks = GdsInstrument.bundleKeywords(
           List(
             buildDoubleS(tcsKeywordsReader.getParallacticAngle
-                          .map(_.map(_.toSignedDoubleDegrees))
+                          .map(_.map(_.toDegrees))
```

Comment:
Why this? I think we explicitly need a signed value

Reply:
Yes, but I made `getParallacticAngle` return an already normalized value.